### ISSUE TITLE
Set the name shown to the user (client) to be the server name specified in the install script

### ISSUE THREAD

**Issue by bluetardis** [trailofbits/algo#491]
### OS / Environment

AWS

### Summary of the problem
During install a server name can be specified in the script. (A query could also be made for a "friendly name".
Regardless of what is entered, the name used is "Algo"  for:
AWS Console
Apple MobileConfig 
(and probably others that I can't test)

### Steps to reproduce the behavior
normal install

### The way of deployment (cloud or local)
cloud

### Expected behavior
use the specified server name or optionally a friendly name/tag

**Comment by dguido** [comment]
Oops

**Comment by bluetardis** [comment]
You guys are fast :-)
Whats the preferred way to donate? Patreon

**Comment by dguido** [comment]
Any way is fine! Thanks for reporting the bug.

**Comment by dguido** [comment]
Can you provide a screenshot of what you are referring to?

**Comment by bluetardis** [comment]
Here are some examples.

## Step1 We do the install and give the server a name.
We override the default name of "Algo" with *testing123*
<img width="800" alt="1 naming-server" src="https://cloud.githubusercontent.com/assets/5024761/25550971/b31932f2-2cc1-11e7-951b-c45bafd9d8de.png">

## Step 2 Here is an example of the name not being used for AWS
We can see that it names the server *Algo*  It does this for all build regardless of the name enterered
![2 bug-example-aws](https://cloud.githubusercontent.com/assets/5024761/25550979/f1e5d36e-2cc1-11e7-9a0c-730c7a548e32.png)


## Step 3 Client implications
At the client end the name shown to the user will be incorrect but its not shows as just Algo.


### Apple MobileConfig
In the username.mobileconfig file, the following lines indicates the expected result (the server name entered in step1).  Currently this needs to be edited manually.
```
            <key>PayloadDisplayName</key>
            <string>testing123</string>
```

### Windows
In the generated windows_username.ps1 file the following lines indicate the expected result (the server name entered in step1).  Currently this needs to be edited manually.
```
Add-VpnConnection -name "testing123" -ServerAddress "12.34.56.78" -TunnelType IKEv2 -AuthenticationMethod MachineCertificate -EncryptionLevel Required
Set-VpnConnectionIPsecConfiguration -ConnectionName "testing123" -AuthenticationTransformConstants GCMAES128 -CipherTransformConstants GCMAES128 -EncryptionMethod AES128 -IntegrityCheckMethod SHA384 -DHGroup ECP256 -PfsGroup none
```

**Comment by bluetardis** [comment]
Let me know if you need more clarification but i figured that was more useful than screen shots from client devices.

**Comment by dguido** [comment]
This makes sense, but we're going to take a look at this after we finish the 1.0 milestone.

**Comment by bkono** [comment]
Found this issue while looking to see if I had missed a config step. It was bugging me, so I went ahead and made initial changes to line up with the output I had hoped to see. Mainly a starting point for a second convenience feature I added for myself, but I kept it separate, just in case this is along the lines of what you were looking to do. If it is, let me know and I'd be happy to clean it up, get some additional testing in and PR it back. https://github.com/trailofbits/algo/compare/master...bkono:feature/server-name-in-mobileconfig
